modules/support-on-various-platforms.adoc

```diff
@@ -0,0 +1,130 @@
+//
+// * support/getting-support-stackrox.adoc
+:_module-type: CONCEPT
+[id="support-on-various-platforms_{context}"]
+= Support on various platforms
+
+[role="_abstract"]
+StackRox support team supports a platform version based on the upstream (or vendor) product’s support lifecycle for that version. When a platform version support reaches its end-of-life (EOL) or is not actively maintained, StackRox no longer support it.
+
+[WARNING]
+====
+StackRox do not support:
+
+* Installing the StackRox Kubernetes Security Platform on Minikube and other similar single-node clusters.
+* Amazon Elastic File System (Amazon EFS), use the Amazon Elastic Block Store (Amazon EBS) with the default GP2 volume type instead.
+* Older CPUs that don’t have the Streaming SIMD Extensions (SSE) 4.2 instruction set, for example Intel processors older than Sandy Bridge and AMD processors older than Bulldozer. Both of these processors were released in 2011.
+====
+
+[id="operating-systems_{context}"]
+== Operating systems
+
+|===
+| Operating system | Version
+
+| http://releases.ubuntu.com/[Ubuntu]
+| 16.04 LTS, 18.04 LTS, and 20.04 LTS with standard or cloud-provider-specific kernel versions.
+
+| https://www.debian.org/releases/[Debian]
+| 9, 10
+
+| https://access.redhat.com/support/policy/updates/errata[Red Hat Enterprise Linux]
+| 7.3  till 7.9, 8.0 and newer
+
+| https://www.centos.org/centos-linux/[CentOS]
+| 7, 8
+
+| https://getfedora.org/en/coreos?stream=stable[Fedora CoreOS]
+| Stable stream 32.20200824.3.0 and newer
+
+| https://www.flatcar-linux.org/releases/[Flatcar Container Linux]
+| 2023.4.0 and newer
+
+| https://cloud.google.com/container-optimized-os/docs/release-notes#current_active_releases[Google COS]
+| 77 and newer
+
+| https://aws.amazon.com/amazon-linux-ami[Amazon Linux]
+| 2
+
+| https://github.com/gardenlinux/gardenlinux[Garden Linux]
+| 27.0 and newer
+|===
+
+[id="container-runtimes_{context}"]
+== Container runtimes
+
+|===
+| Container runtimes | Version
+
+| https://docs.docker.com/engine/release-notes/[Docker]
+| 17.03 and newer
+
+| https://github.com/cri-o/cri-o/releases[CRI-O] and https://github.com/opencontainers/runc/releases[runC]
+| -
+|===
+
+[NOTE]
+====
+CRI-O and runC support is available starting from {product-title} version 2.5.31.0.
+====
+
+[id="container-orchestrators-and-platforms_{context}"]
+== Container orchestrators and platforms
+
+|===
+| Container orchestrators and platforms | Version
+
+| https://kubernetes.io/docs/setup/release/notes/[Kubernetes]^1^
+| 1.15 and newer
+
+| https://access.redhat.com/support/policy/updates/openshift[{ocp}]
+| 3.10, 3.11, 4.1 and newer
+
+| https://docs.d2iq.com/mesosphere/dcos/services/kubernetes/[DC/OS Kubernetes]^2^
+| 2.0.0 and newer
+|===
+
+[NOTE]
+====
+
+. StackRox support new versions of Kubernetes within 3 months of their open-source general-availability release.
+You may need to upgrade the {product-title} to get support for the latest Kubernetes versions.
```

Comment:
s/You may need to upgrade the {product-title}/You might need to upgrade {product-title}